ff/src/lib.rs

```diff
@@ -163,6 +167,7 @@ pub trait PrimeFieldRepr:
     }
 
     /// Reads a big endian integer into this representation.
+    #[cfg(feature = "std")]
     fn read_be<R: Read>(&mut self, mut reader: R) -> io::Result<()> {
```

Comment:
This API will be replaced with `no_std`-compatible `from_bytes` (matching the `bls12_381` and `jubjub` APIs) when we remove `PrimeFieldRepr`.